chimba/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "chimba",
```

Comment:
If this is indeed a testnet, then this should be 'chimbatestnet' and this should all be in the /testnets/ folder.